Resources/doc/overview.rst

```diff
@@ -8,7 +8,19 @@ This bundle is available on Packagist_. You can install it using Composer:
 
 .. code-block:: bash
 
-    $ composer require friendsofsymfony/http-cache-bundle:~1.0
+    $ composer require friendsofsymfony/http-cache-bundle:~2.0
+
+Note that the FOSHttpCache_ library needs a ``psr/http-message-implementation``
+and ``php-http/adapter-implementation``. If your project does not contain one,
+composer will complain that it did not find ``psr/http-message-implementation``.
```

Comment:
this is copy-pasted from FOSHttpCache but i think its better to duplicate than to reference at this point.